modin/pandas/test/test_series.py

```diff
@@ -1141,6 +1141,20 @@ def test_between_time():
     )
 
 
+def test_time_ops():
+    # Make a pandas.core.indexes.timedeltas.TimedeltaIndex
+    deltas = pd.to_timedelta([1], unit="h")
+    modin_series = pd.Series(np.datetime64("2000-12-12")) + deltas
+    pandas_series = pandas.Series(np.datetime64("2000-12-12")) + deltas
+
+    df_equals(modin_series, pandas_series)
```

Comment:
```suggestion
    test_series = create_test_series(np.datetime64("2000-12-12"))
    eval_general(*test_series, lambda s: s + deltas)
```

Reply:
Wow this is super clean, I'll try to make use of these idiom properly in the future.